ts/components/core/variables/IOColors.ts

```diff
@@ -5,27 +5,114 @@ function asIOColors<T extends { [key: string]: ColorValue }>(arg: T): T {
+  itemBorderDefaultColor:
+    "#5F6F82" /* from variables.ts, similar to milderGray, ALIAS TOKEN */,
+  milderGray: "#5F6F80" /* from variables.ts */,
+  darkerGray: "#4B5C6F" /* from variables.ts */,
+  bluegrey: "#475A6D",
+  brandDarkGray: "#475A6D" /* from variables.ts, duplicate of bluegrey */,
+  textColor:
+    "#475A6D" /* from variables.ts, duplicate of bluegrey, ALIAS TOKEN */,
+  topTabBarTextColor:
+    "#475A6D" /* from variables.ts, duplicate of bluegrey, ALIAS TOKEN */,
+  unselectedColor:
+    "#475A6D" /* from variables.ts, duplicate of bluegrey, ALIAS TOKEN */,
+  bluegreyDark: "#17324D",
+  brandDarkestGray:
+    "#17324D" /* from variables.ts, duplicate of bluegreyDark */,
+  cardFontColor:
+    "#17324D" /* from variables.ts, duplicate of bluegreyDark and brandDarkestGray, ALIAS TOKEN */,
+  textColorDark: `rgb(28, 28, 30)` /* found in various JEST tests */,
+  black: "#000000",
+  colorBlack: "#000000" /* from variables.ts, duplicate of black */,
+  footerShadowColor:
+    "#000000" /* from variables.ts, duplicate of black, ALIAS TOKEN */,
+  brandPrimary: "#0066CC" /* from variables.ts */,
+  topTabBarActiveTextColor:
+    "#0066CC" /* from variables.ts, duplicate of brandPrimary, ALIAS TOKEN */,
   blue: "#0073E6",
+  selectedColor:
+    "#0073E6" /* from variables.ts, duplicate of blue, ALIAS TOKEN */,
+  contentPrimaryBackground:
+    "#0073E6" /* from variables.ts, duplicate of blue, ALIAS TOKEN */,
+  ListSelectionBarColor: "#007AFF" /* from ListSelectionBar JEST test */,
+  textLinkColor:
+    "#0073E6" /* from variables.ts, duplicate of blue, ALIAS TOKEN */,
+  checkBoxColor: "#039BE5" /* from CheckBox component */,
+  brandPrimaryLight: "#99CCFF" /* from variables.ts */,
+  noCieButton:
+    "#789CCD" /* Background of half-disabled noCIE CTA, from LandingScreen.tsx */,
+  featuredCardShadow: "#00274E" /* from FeaturedCard component */,
+  activeBonus: "#2C489D" /* from relative component (BonusVacanze) */,
+  cgnDiscount: "#9B5897" /* from CgnDiscountValueBox */,
   aqua: "#00C5CA",
+  brandHighlight: "#00C5CA" /* from variables.ts, duplicate of aqua */,
+  brandHighLighter: "#00CDD2" /* from variables.ts */,
+  toastColor: "#C1F4F2" /* from variables.ts, ALIAS TOKEN */,
+  badgeYellow: "#FFC824" /* from variables.ts, ALIAS TOKEN */,
+  cgnDiscountDetailBg: "#EB9505" /* from relative component */,
+  cardExpiredTextColor: "#FF0000" /* from variables.ts, ALIAS TOKEN */,
+  calendarExpirableColor: "#D0021B" /* from variables.ts, ALIAS TOKEN */,
+  itemTest: "#ED1727" /* from relative JEST snap test */,
   red: "#C02927",
+  brandDanger: "#CC3333" /* from variables.ts */,
+  brandSuccess: "#007005" /* from variables.ts */,
+  orange: "#EA7614",
   greyGradientTop: "#475A6D",
   greyGradientBottom: "#42484F",
   yellowGradientTop: "#FEC709",
-  yellowGradientBottom: "#FAA01F",
-  orange: "#EA7614"
+  yellowGradientBottom: "#FAA01F"
+});
+
+export const IOColorGradients = asIOColorGradients({
```

Comment:
As it appens with `IOColors` what about exporting `IOColorGradientsType`

```typescript
export type IOColorGradientType = keyof typeof IOColorGradients;
```

Reply:
I've already added the line in the [PR #4076](https://github.com/pagopa/io-app/pull/4076). [You can view the specific line here](https://github.com/pagopa/io-app/pull/4076/commits/8f7b06a6c9ad9b2a9e8fbf5c3b3bffe3c15c9dd1).

I haven't added it in this PR because there was no need in this specific branch.